### PR TITLE
1.0 rc

### PR DESCRIPTION
**Pull request by andyislegend**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,9 +209,208 @@ public void cvurl() {
...
     }
 ```
+And another example - due to underlying Mapper (by default it's Jackson) You can send use Java objects as a Request body without a hassle
+```java
+ public UserCreatedResponseDto createUser(UserDto userDto) {
+        return cVurl.POST(HOST + USERS)
+                .body(userDto)
+                .header(HttpHeader.CONTENT_TYPE, MIMEType.APPLICATION_JSON)
+                .build()
+                .asObject(UserCreatedResponseDto.class, HttpStatus.CREATED);
```
Comment (yorekKap) [inline README.md]: this example is not correct - method asObject(class, statusCode) returns optional.

File: README.md [hunk 2]
```diff
@@ -209,9 +209,208 @@ public void cvurl() {
...
+```
+We do not fancy such verbosity.
+That's why we do all stuff for you under the hood. What You really use is just this simple and nice-looking (fingers crossed) API.
+
+###### cVurl formdata example
+```java
+ public User createUserFromFormUrlencoded(Map<String, String> userMap) {
+        return cVurl.post("https://...")
+                .formData(userMap)
+                .header(HttpHeader.CONTENT_TYPE, MIMEType.APPLICATION_FORM)
```
Comment (yorekKap) [inline README.md]: there is no  need to set CONTENT_TYPE header here, formData method does it implicitly.

Comment (andyislegend) [reply README.md]: but you do that in examples. Will refactor

File: README.md (hunk 1 above)
Comment (andyislegend) [reply README.md]: Thx for noticing, my fault

File: README.md (hunk 2 above)
Comment (yorekKap) [reply README.md]: sorry, my bad, will change examples

**Pull request merged**